sdk/spring/spring-security-azure-oauth2-client/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure.spring</groupId>
+  <artifactId>spring-security-azure-oauth2-client</artifactId>
```

Comment:
@JonathanGiles are we OK with calling this Oauth2?

Reply:
Thanks for the call out - I'll discuss internally today.

Reply:
FYI: It's just an extension of [spring-security-oauth2-client].



[spring-security-oauth2-client]: https://repo1.maven.org/maven2/org/springframework/security/spring-security-oauth2-client/